django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -195,4 +195,12 @@ Requires core.js, SelectBox.js and addevent.js.
         }
     };
 
+    addEvent(window, "load", function(e) {
+        $('select.selectfilter, select.selectfilterstacked').each(function() {
+            var $el = $(this),
```

Comment:
It looks to me like we don't configure this rule so either form is accepted now? Are you advocating switching to this style? I'd rather match the current style and switch later instead of introducing inconsistencies, but let me know your thoughts.